pkg/cloud/services/ec2/bastion_test.go

```diff
@@ -327,6 +323,10 @@ func TestServiceReconcileBastion(t *testing.T) {
 						Name:   aws.String("description"),
 						Values: aws.StringSlice([]string{ubuntuImageDescription}),
 					},
+					{
+						Name:   aws.String("owner-id"),
+						Values: aws.StringSlice([]string{ubuntuOwnerID}),
+					},
```

Comment:
Please add a test so we see that he right owner is applied if necessary.

Reply:
A test case has been added for us-gov partition.